lib/test_cases/function/pull_request_file.js

```diff
@@ -409,6 +409,241 @@ var codeCensorPullRequestFile = function(){
         return result;
     }
 
+    /**
+     * Checks package information is present or not.
+     */
+    function pullRequestFileCheckedSecuredInput(prFile) {
+        let result = [];
+        if (prFile.type === 'js') {
+            let targetPrLine = undefined;
+            let assigned_variable = undefined;
+            prFile.lines.forEach(function (currentPrLine) {
+                if (currentPrLine instanceof PrLine) {
+                    const pattern = new RegExp("text\\(\\)");
+                    let content = currentPrLine.content;
+                    let form_keywords = ['form', 'input', 'select', 'textarea'];
+                    if (pattern.test(content)) {
+                        form_keywords.forEach(function (word, index) {
+                            if (content.includes(word)) {
+                                targetPrLine = currentPrLine;
+                                if (!content.includes("Drupal.checkPlain")) {
+                                    assigned_variable = content.split("=")[0].trim().split(" ").pop();
+                                }
+                            }
+                        });
+                    }
+                    if (typeof (assigned_variable) !== 'undefined') {
+                        if (content.includes(assigned_variable) && content.includes("Drupal.checkPlain")) {
```

Comment:
content being used over here for validation looks like will be working on currentPrLine content and not the subsequent ones.. while we will generally have the checks in next line or subsequent lines. Kindly validate this scenario.